src/events/aws_sdk.js

```diff
@@ -230,6 +230,11 @@ const SQSEventCreator = {
             });
             break;
         case 'sendMessageBatch':
+            if (response.data.Successful && response.data.Successful.length > 0) {
```

Comment:
Check that response.data exists

Reply:
It can't be since it's part of the HTTP Response object (AWS SDK). We really want it to break in case of such a major API change rather than ignoring it. either way it'll be part of the Trace Exceptions[] and won't break the user code